Dockerfile

```diff
@@ -3,6 +3,20 @@
 # tools/build-release-tarball to generate a production release tarball
 # from the provided Git ref.
 FROM ubuntu:24.04 AS base
+# Install required packages
+RUN apt-get update && apt-get install -y \
+    logrotate \
+    cron \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
We install dependencies via `puppet`, you shouldn't need to add anything here. I expect these files are already present, and the actual bug is needing to actually run `logrotate` via `supervisord` (like how we do cron), since it likely is installed but not running.